object_database/frontends/service_entrypoint.py

```diff
@@ -50,19 +49,23 @@ def main(argv):
         )
 
     def dbConnectionFactory():
-        return connect(parsedArgs.host, parsedArgs.port)
+        return connect(parsedArgs.host, parsedArgs.port, parsedArgs.serviceToken)
 
     setCodebaseInstantiationDirectory(parsedArgs.sourceDir)
 
     try:
-        manager = ServiceWorker(dbConnectionFactory, parsedArgs.instanceid, parsedArgs.storageRoot)
+        manager = ServiceWorker(
+            dbConnectionFactory, parsedArgs.instanceid,
+            parsedArgs.storageRoot, parsedArgs.serviceToken
+        )
 
         manager.runAndWaitForShutdown()
 
         return 0
-    except:
+    except Exception:
```

Comment:
@braxtonmckee a naked `except` catches both `Exception` and `KeyboardInterrupt`, but I figured we only want to be catching Exceptions in most of these cases.